src/main/java/graphql/testrunner/resource/TestRunnerResource.java

```diff
@@ -0,0 +1,31 @@
+package graphql.testrunner.resource;
+
+import java.util.Map;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import graphql.testrunner.service.TestRunnerService;
+
+import static java.util.Objects.nonNull;
+
+@RestController
+public class TestRunnerResource {
+
+    @Autowired
+    private TestRunnerService testRunnerService;
+
+    @PostMapping("/test-runner")
+    public void runTest(@RequestBody Map<String, String> commitHash) {
+        if (nonNull(commitHash) || !commitHash.containsKey("commit_hash"))
+            testRunnerService.runTest(commitHash.get("commit_hash"));
```

Comment:
At some point I imagine that handling this request might need to hand off work to a dedicated thread pool and go async. I expect that even checking out the graphql-java repo might be slow enough that we'll want to start planning for that now.